.github/workflows/python-package-conda.yml

```diff
@@ -0,0 +1,46 @@
+name: Conda-based pytest
+
+on: [push, pull_request]
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      max-parallel: 5
+      matrix:
+        python-version: [3.6, 3.9]
+        slycot: ["", "conda"]
+        array-and-matrix: [0]
+        include:
+          - python-version: 3.9
+            slycot: conda
+            array-and-matrix: 1
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        # Set up conda
+        echo $CONDA/bin >> $GITHUB_PATH
+        conda create -q -n test-environment python=${{matrix.python-version}} 
+        source $CONDA/bin/activate test-environment
+
+        # Set up (virtual) X11
+        sudo apt install -y xvfb
+
+        # Install test tools
+        conda install pip coverage pytest 
+        conda install -c conda-forge pytest-xvfb pytest-cov
```

Comment:
coverage not reported to coveralls yet

Reply:
I think we need to merge the PR before this is run and (hopefully) at that point things start to show up in coveralls?

Reply:
`coverage` != `coveralls`.

See https://coveralls-python.readthedocs.io/en/latest/usage/configuration.html#github-actions-gotcha and my previously linked workflow draft for Slycot. (Much more complicated than yours! :( )